example/pubspec.yaml

```diff
@@ -37,7 +37,7 @@ dependencies:
   reactive_date_time_picker: ^0.1.3
 
 dev_dependencies:
-  pedantic: ^1.11.1
+  lints: ^1.0.1
```

Comment:
Very good, thanks @kuhnroyal for updating to the new package and migrate the project to the new lint. :)